docs/language/support/reusables/frameworks.rst

```diff
@@ -18,6 +18,46 @@ C# built-in support
    NHibernate, Database ORM
    WinForms, User interface
 
+Go built-in support
+================================
+
+.. csv-table::
+   :header-rows: 1
+   :class: fullWidthTable
+   :widths: auto
+
+   Name, Category
+   SendGrid, Email library
+   json-iterator, Serialization
+   Gin, Web framework
+   glog, Logging library
+   klog, Logging library
+   go-restful, Web application framework
+   Logrus, Logging library
+   Macaron, Web framework
+   mux, http request router and dispatcher
+   mongo, Database
+   Squirrel, Database
+   pg, Database
+   GORM, Database
+   sqlx, Database
+   go-sh, Utility library
+   golang.org/x/crypto/ssh, Network communicator
+   golang.org/x/net/websocket, Network communicator
+   Gorilla websocket, Network communicator
+   ws, Network communicator
+   nhooyr.io/websocket, Network communicator
+   GoWebsocket, Network communicator
+   ws, Network communicator
+   https://github.com/antchfx/xpath, Xpath library
```

Comment:
```suggestion
   https://github.com/antchfx/xpath, XPath library
```
Isn't it usually capitalized like this: `XPath`?
See the other instances below.